packages/forms-web-app/src/config.js

```diff
@@ -43,5 +34,6 @@ module.exports = {
   featureFlag: {
     googleTagManager: process.env.FEATURE_FLAG_GOOGLE_TAG_MANAGER === 'true',
     usePrivateBetaV1RoutesOnly: process.env.PRIVATE_BETA_V1_ROUTES_ONLY === 'true',
+    useRedisSessionStore: process.env.FEATURE_SESSION_STORE === 'true',
```

Comment:
The env var should be `FEATURE_REDIS_SESSION_STORE`

Reply:
done